kythe/go/extractors/config/runextractor/README.md

```diff
@@ -30,13 +30,14 @@ Use:
 
 Because `runextractor` is invoked in a manner not conducive to cleanly passing
 commandline flags, there is non-trivial setup done with environment variables.
-When calling `runextractor`, here are the relevant environment variables:
+When calling `runextractor`, here are the required environment variables:
 
-* **KYTHE_CORPUS**: The corpus set for extracted files.
-* **KYTHE_ROOT_DIRECTORY**: The root path for file input to be extracted.
-* **KYTHE_OUTPUT_DIRECTORY**: The path for storing output.
+* **KYTHE_ROOT_DIRECTORY**: The absolute path for file input to be extracted.
+* **KYTHE_OUTPUT_DIRECTORY**: The absolute path for storing output.
+* **KYTHE_CORPUS**: The corpus label for extracted files.  If not set it
```

Comment:
If it has a default, it's not really required, right?

Reply:
Yea this was an explanation of the underlying behavior of the java and other extractors, guess it isn't necessary to put here because we force caller to set it.